src/fights.ts

```diff
@@ -1428,6 +1428,62 @@ const freeFightSources = [
     },
     true
   ),
+  // Use NC forcers to complete shadow quests
+  new FreeFight(
+    () => {
+      if (!have($item`closed-circuit pay phone`)) return false;
+      if (globalOptions.prefs.yachtzeechain) return false; // NCs are better when yachtzeeing, probably
+
+      if (get("rufusQuestType") === "items") {
+        return false; // NCs don't help with item quests
+      }
+
+      // Check NC-forcers
+      if (get("encountersUntilSRChoice") === 0) {
+        // NC is forced natively
+        return true;
+      }
+
+      if (have($item`Clara's bell`) && !get("_claraBellUsed")) {
+        return true;
+      }
```

Comment:
Note that we sometimes intentionally save the clara's bell for the volcano (if the best quest for the volcoino requires us to force an NC)

Reply:
The Volcano code runs in DailySetup, before free fights happen, so I think we're all good on this part.